### ISSUE TITLE
How are the hyperparameters tuned?

### ISSUE THREAD

**Issue by jry01** [WonwoongCho/GDWCT#6]
Thanks for the great work! My question is that how the hyperparameters are set as it is a complex network to train?

**Comment by WonwoongCho** [comment]
Hi hongboyi, thank you for your attention to GDWCT.

As written in the paper, the hyperparameters are empirically set, after being quantitatively as well as qualitatively experimented. 

Additionally, except for the hyperparameters of the regularizations and the number of groups, which our paper proposed, we referred to other baseline works, such as MUNIT and DRIT.  

Based on our experiences, the hyperparameters of pixel-level reconstruction losses are directly relevant to the translated-output quality, hence we recommend to set those hyperparameters larger than that of gan loss.

**Comment by jry01** [comment]
Thank you for your reply.
